docs/database/schema-alternative.sql

```diff
@@ -0,0 +1,101 @@
+--
+-- Copyright (C) 2016 Red Hat, Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--         http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+CREATE EXTENSION hstore;
+
+-- ==== Instances ====================================================================
+
+-- An action instance is the single entry point in this sub domain. It is referenced
+-- by an integration and has all configuration needed for creating an instance of
+-- a fully configured Camel connector
+CREATE TABLE action_instance
```

Comment:
I also opt for going on and keep the configuration in the DB for now.

One solution could be, that a expert dev only edits `template yamls` and the final yamls are generated artifacts which e.g. could be automatically regenerated . In this templates there are only the _reference to the configuration parameters_ stored (e.g the value could be ignored or some variable ref could be used). During the render process the DB is contacted and the values are filled in. This would also clearly separate what can be done by an expert dev (change the template) and by the cit user, too (define values to be filled in).